grpc-gcp/package.json

```diff
@@ -34,20 +34,19 @@
   "engines": {
     "node": ">=6"
   },
-  "dependencies": {
-    "grpc": "^1.16.0",
-    "protobufjs": "^6.8.8"
-  },
+  "dependencies": {},
   "devDependencies": {
+    "@grpc/grpc-js": "^0.7.3",
```

Comment:
With the changes I made, `grpc-js` is only needed for the TypeScript build. `grpc-js` isn't used at runtime because the user of `grpc-gcp` needs to pass in a reference to a gRPC implementation.

Reply:
On second thought, it should be in `dependencies` because it's used in some of the types exported by grpc-gcp. I've made the change.